### ISSUE TITLE
Make recipe00 less error-prone

### ISSUE THREAD

**Issue by MaxGhenis** [PSLmodels/Tax-Calculator#2263]
Following up on #2259, in which I tried recipe00 and hit an error because I downloaded https://pslmodels.github.io/Tax-Calculator/reformA.json, which is actually an HTML file not a JSON file (I mistakenly attributed the error to comments in that issue, so it's closed).

In https://github.com/PSLmodels/Tax-Calculator/issues/2259#issuecomment-473581235 I described my interpretation as to the recipe00 steps, and how to simplify it for new users. Adapting that with some new information to motivate and describe this issue:

The intended way of running recipe00 seems to be:
1. Copy the contents of https://pslmodels.github.io/Tax-Calculator/reformA.json.html (linked from the cookbook) from the webpage into a text editor, and save that as `reformA.json` in an `ingredients` subfolder.
2. Copy the contents of https://pslmodels.github.io/Tax-Calculator/recipe00.py.html from the webpage into a text editor, and save that as `recipe00.py` (`wget` will produce the same problem for the `py` file).
3. Run `python recipe00.py > recipe00.out`.

I think these changes would be simpler to users (especially important if this intends to be their very first usage of taxcalc), in order of significance:
1. Document the above steps in the cookbook so users know exactly what to do.
2. Remove the `.json.html` files and point instead to the raw versions hosted on GitHub, like https://raw.githubusercontent.com/PSLmodels/Tax-Calculator/master/docs/cookbook/ingredients/reformA.json. At this point, a user can use `wget` to pull down a file without copy-pasting or removing headers, and it also reduces code duplication (e.g. https://github.com/PSLmodels/Tax-Calculator/commit/7847010b28bbd4faad08276f3067ba8f2a986f4e required modifying both files in parallel).
3. Adapt the code to pull the reform file from the web, to remove the need for the `ingredients` folder and downloading the reform file.
4. Put it all in a self-contained notebook that "just works" (only requires `taxcalc` to be installed). I've done this at https://github.com/MaxGhenis/taxcalc-notebooks/blob/master/recipes/recipe00.ipynb.

**Comment by martinholmer** [comment]
@MaxGhenis, after the merge of PR #2265, I can do this on my Mac computer (that has `curl` installed, but not `wget`):
```
iMac:nowhere mrh$ pwd
/Users/mrh/nowhere

iMac:nowhere mrh$ ls

iMac:nowhere mrh$ conda install -c PSLmodels behresp --yes
Collecting package metadata: done
Solving environment: done

## Package Plan ##

  environment location: /Users/mrh/anaconda3

  added / updated specs:
    - behresp


The following NEW packages will be INSTALLED:

  behresp            PSLmodels/osx-64::behresp-0.7.0-py36_0
  taxcalc            PSLmodels/osx-64::taxcalc-1.1.0-py36_0


Preparing transaction: done
Verifying transaction: done
Executing transaction: done

iMac:nowhere mrh$ curl -O "https://pslmodels.github.io/Tax-Calculator/cookbook/recipe0[0-4].{py,res}" -O "https://pslmodels.github.io/Tax-Calculator/cookbook/reform[A-B].json" -O "https://pslmodels.github.io/Tax-Calculator/cookbook/test_recipes.py"

[1/10]: https://pslmodels.github.io/Tax-Calculator/cookbook/recipe00.py --> recipe00.py
--_curl_--https://pslmodels.github.io/Tax-Calculator/cookbook/recipe00.py
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  3601  100  3601    0     0  13537      0 --:--:-- --:--:-- --:--:-- 13588

[2/10]: https://pslmodels.github.io/Tax-Calculator/cookbook/recipe00.res --> recipe00.res
--_curl_--https://pslmodels.github.io/Tax-Calculator/cookbook/recipe00.res
100  6730  100  6730    0     0   205k      0 --:--:-- --:--:-- --:--:--  205k

[3/10]: https://pslmodels.github.io/Tax-Calculator/cookbook/recipe01.py --> recipe01.py
--_curl_--https://pslmodels.github.io/Tax-Calculator/cookbook/recipe01.py
100  2658  100  2658    0     0  12305      0 --:--:-- --:--:-- --:--:-- 12305

[4/10]: https://pslmodels.github.io/Tax-Calculator/cookbook/recipe01.res --> recipe01.res
--_curl_--https://pslmodels.github.io/Tax-Calculator/cookbook/recipe01.res
100  2140  100  2140    0     0  52195      0 --:--:-- --:--:-- --:--:-- 52195

[5/10]: https://pslmodels.github.io/Tax-Calculator/cookbook/recipe02.py --> recipe02.py
--_curl_--https://pslmodels.github.io/Tax-Calculator/cookbook/recipe02.py
100  2546  100  2546    0     0  43896      0 --:--:-- --:--:-- --:--:-- 43896

[6/10]: https://pslmodels.github.io/Tax-Calculator/cookbook/recipe02.res --> recipe02.res
--_curl_--https://pslmodels.github.io/Tax-Calculator/cookbook/recipe02.res
100  4979  100  4979    0     0  21097      0 --:--:-- --:--:-- --:--:-- 2781k

[7/10]: https://pslmodels.github.io/Tax-Calculator/cookbook/recipe03.py --> recipe03.py
--_curl_--https://pslmodels.github.io/Tax-Calculator/cookbook/recipe03.py
100  1431  100  1431    0     0  25553      0 --:--:-- --:--:-- --:--:-- 25553

[8/10]: https://pslmodels.github.io/Tax-Calculator/cookbook/recipe03.res --> recipe03.res
--_curl_--https://pslmodels.github.io/Tax-Calculator/cookbook/recipe03.res
100  1047  100  1047    0     0  16107      0 --:--:-- --:--:-- --:--:-- 16107

[9/10]: https://pslmodels.github.io/Tax-Calculator/cookbook/recipe04.py --> recipe04.py
--_curl_--https://pslmodels.github.io/Tax-Calculator/cookbook/recipe04.py
100  3350  100  3350    0     0  57758      0 --:--:-- --:--:-- --:--:-- 57758

[10/10]: https://pslmodels.github.io/Tax-Calculator/cookbook/recipe04.res --> recipe04.res
--_curl_--https://pslmodels.github.io/Tax-Calculator/cookbook/recipe04.res
100   423  100   423    0     0  10317      0 --:--:-- --:--:-- --:--:-- 10317

[1/2]: https://pslmodels.github.io/Tax-Calculator/cookbook/reformA.json --> reformA.json
--_curl_--https://pslmodels.github.io/Tax-Calculator/cookbook/reformA.json
100   598  100   598    0     0  19290      0 --:--:-- --:--:-- --:--:-- 19290

[2/2]: https://pslmodels.github.io/Tax-Calculator/cookbook/reformB.json --> reformB.json
--_curl_--https://pslmodels.github.io/Tax-Calculator/cookbook/reformB.json
100   185  100   185    0     0   6166      0 --:--:-- --:--:-- --:--:--  6166
100  1814  100  1814    0     0  20613      0 --:--:-- --:--:-- --:--:-- 20613

iMac:nowhere mrh$ ls
recipe00.py	recipe01.res	recipe03.py	recipe04.res	test_recipes.py
recipe00.res	recipe02.py	recipe03.res	reformA.json
recipe01.py	recipe02.res	recipe04.py	reformB.json

iMac:nowhere mrh$ python test_recipes.py 
2019-03-20 14:20:14
recipe00.py PASS
recipe01.py PASS
recipe02.py PASS
recipe03.py PASS
recipe04.py PASS
2019-03-20 14:22:44

iMac:nowhere mrh$ 
```

**Comment by martinholmer** [comment]
Given that there has been no discussion in issue #2263 over the past eleven days, I'm going to close it.
@MaxGhenis, if you want to discuss these topics further, please open a new issue.
